### ISSUE TITLE
Would it be possible and interesting to group some menus?

### ISSUE THREAD

**Issue by jvesouza** [microsoft/vscode#192092]
Type: <b>Feature Request</b>

When I press shift+f10 to access the available menus, I sometimes need to press down arrow or up arrow until I find the desired menu. 
I think this could be improved if more menus were grouped together like some already existing groups.
To give an example, the menus dealing with go to could be grouped as was done for peak.

VS Code version: Code - Insiders 1.82.0-insider (f1302be1e67e3af5fbeb8bbb2ea784de7bc96150, 2023-09-01T11:05:38.381Z)
OS version: Linux x64 6.4.13-200.fc38.x86_64
Modes:

**Comment by jrieken** [comment]
> I think this could be improved if more menus were grouped together like some already existing groups.

Yes and no. The downside is that it breaks muscle memory and forces users to click more often. When we did the changes for peek many folks didn't like them. It's surprising how sensitive users are to changes in the context menu so any major change needs to be for good reasons and validated
